src/lib/GRPCClient.ts

```diff
@@ -0,0 +1,68 @@
+import { loadSync, PackageDefinition } from '@grpc/proto-loader'
+import { Client, credentials, loadPackageDefinition } from 'grpc'
+
+export class GRPCClient {
+	private packageDefinition: PackageDefinition
+	private client: Client
+	private listNameMethods: any[]
+
+	constructor(
+		protoPath: string,
+		packageName: string,
+		service: string,
+		host: string,
+		options = {} as any,
```

Comment:
Can we get a type instead of `any` or can you explain why `any`?
I would replace to something like `Partial<IOptionsThatWeManage>`?

Reply:
I'll make it the proto-loader.Options. to expose all available Options.